administrator/language/en-GB/en-GB.plg_quickicon_joomlaupdate.ini

```
@@ -8,8 +8,8 @@ PLG_QUICKICON_JOOMLAUPDATE_CHECKING="Checking Joomla ..."
 PLG_QUICKICON_JOOMLAUPDATE_ERROR="Unknown Joomla ..."
 PLG_QUICKICON_JOOMLAUPDATE_GROUP_DESC="The group of this plugin (this value is compared with the group value used in <strong>Quick Icons</strong> modules to inject icons)."
 PLG_QUICKICON_JOOMLAUPDATE_GROUP_LABEL="Group"
-PLG_QUICKICON_JOOMLAUPDATE_UPDATEFOUND="Joomla %s, Update now!"
+PLG_QUICKICON_JOOMLAUPDATE_UPDATEFOUND="Joomla %s Update now!"
 PLG_QUICKICON_JOOMLAUPDATE_UPDATEFOUND_BUTTON="Update Now"
-PLG_QUICKICON_JOOMLAUPDATE_UPDATEFOUND_MESSAGE="Joomla %s is available:"
+PLG_QUICKICON_JOOMLAUPDATE_UPDATEFOUND_MESSAGE="Joomla %s is available"
```

This is now a sentence so should end with a full stop.

```suggestion
PLG_QUICKICON_JOOMLAUPDATE_UPDATEFOUND_MESSAGE="Joomla %s is available."
```